src/extension.ts

```diff
@@ -84,7 +84,7 @@ export async function activate(context) : Promise<extensionapi.ExtensionAPI> {
     const completionProvider = new HelmTemplateCompletionProvider();
     const completionFilter = [
         "helm",
-        {language: "yaml", pattern: "**/templates/*.yaml"},
+        // templates/*.yaml should be helm templates, so we needn't select them
```

Comment:
Do we need to keep this commented-out line?

Reply:
removed